.storybook/main.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   addons: [
     '@storybook/addon-knobs',
     '@storybook/addon-backgrounds',
-    '@storybook/addon-docs'
+    '@storybook/addon-docs',
+    'storybook-addon-designs/register'
```

Comment:
The `/register` part is not required.
```suggestion
    'storybook-addon-designs'
```